src/initialData.ts

```diff
@@ -2,32 +2,64 @@ const initialData: InitialData = {
   tasks: {
     "task-1": {
       id: "task-1",
-      content: "Take out the garbage",
+      content: "Hover over ☰ next to Task App on nav bar to add new column",
       complete: false,
     },
     "task-2": {
       id: "task-2",
-      content: "Watch my favorite show",
+      content: "Click on column title text to edit column title",
+      complete: false,
+    },
+    "task-3": {
+      id: "task-3",
+      content: "Hover over individual task to edit or delete task",
```

Comment:
I would say "to view more options" instead of "to edit or delete task", since hovering doesn't trigger edit or delete.

Reply:
done